src/doc_scan_service/session/retrieve/task.response.js

```diff
@@ -0,0 +1,105 @@
+'use strict';
+
+const Validation = require('../../../yoti_common/validation');
+const GeneratedCheckResponse = require('./generated.check.response');
+const GeneratedTextDataCheckResponse = require('./generated.text.data.check.response');
+const GeneratedMedia = require('./generated.media');
+const DocScanConstants = require('../../doc.scan.constants');
+const { YotiDate } = require('../../../data_type/date');
+
+class TaskResponse {
+  constructor(task) {
+    Validation.isString(task.id, 'id', true);
+    this.id = task.id;
+
+    Validation.isString(task.state, 'state', true);
+    this.state = task.state;
+
+    Validation.isString(task.created, 'created', true);
+    this.created = task.created;
+
+    Validation.isString(task.last_updated, 'last_updated', true);
+    this.lastUpdated = task.last_updated;
```

Comment:
These are set below, and so these lines can be removed (must be carry over where we decided on using dates)

Reply:
Fixed in 51076f8